tests/unit/test_schema.py

```diff
@@ -510,9 +510,18 @@ def test___hash__not_equals(self):
 
     def test___repr__(self):
         field1 = self._make_one("field1", "STRING")
-        expected = "SchemaField('field1', 'STRING', 'NULLABLE', None, (), ())"
+        expected = "SchemaField('field1', 'STRING', 'NULLABLE', None, (), None)"
         self.assertEqual(repr(field1), expected)
 
+    def test___repr__evaluable(self):
+        field = self._make_one("field1", "STRING", "REQUIRED", "Description")
+        field_repr = repr(field)
+        SchemaField = self._get_target_class()  # needed for eval  # noqa
+
+        evaled_field = eval(field_repr)
+
+        assert field == evaled_field
+
```

Comment:
For completeness, do we need a round-trip test for the case where there is a policy tuple?

Reply:
Would not hurt, I'll add it.

Reply:
Guess what, that extra test actually revealed deeper issues in the `repr()` of `SchemaField` and `PolicyTagList`.

Reply:
W00t!